boards/arm/adafruit_feather_nrf52840/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# Adafruit Feather nRF52840 Express board configuration
+
+# Copyright (c) 2020 Tobias Svehagen
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ADAFRUIT_FEATHER_NRF52840
+	bool "Feather nRF52840 Express Board"
```

Comment:
Using "Adafruit Feather nRF52840 Express" as this prompt would make it more consistent with what is used for most of other boards.  The adafruit_feather_stm32f405 board seems to be rather an exception to the rule in this regard that should be corrected perhaps.

Reply:
Done